File: ios/Khrysalis Template/com/lightningkite/khrysalistemplate/vg/DrawableDemoVG.shared.swift
//Package: com.lightningkite.khrysalistemplate.vg
//Converted using Khrysalis2

import Foundation
import Khrysalis
import RxSwift
import RxRelay
import KhrysalisMaps



public class DrawableDemoVG: ViewGenerator {
    
    
    override public var title: String {
        get {
            return "Drawable Demo"
        }
    }
    
    override public func generate(dependency: ViewDependency) -> View {
        var xml = DrawableDemoXml()
        var view = xml.setup(dependency)
        return view
    }
    override public func generate(_ dependency: ViewDependency) -> View {
        return generate(dependency: dependency)
    }
    
    override public init() {
        super.init()
    }
}
 

File: android/src/main/java/com/lightningkite/khrysalistemplate/vg/DrawableDemoVG.shared.kt
package com.lightningkite.khrysalistemplate.vg

import android.view.View
import com.lightningkite.khrysalis.views.ViewDependency
import com.lightningkite.khrysalis.views.ViewGenerator
import com.lightningkite.khrysalistemplate.layouts.DrawableDemoXml

class DrawableDemoVG() : ViewGenerator() {
    override val title: String get() = "Drawable Demo"

    override fun generate(dependency: ViewDependency): View {
        val xml = DrawableDemoXml()
        val view = xml.setup(dependency)

        return view
    }
}
